docs/products/kafka/howto/enable-kafka-tiered-storage.rst

```diff
@@ -0,0 +1,92 @@
+Enable tiered storage for Aiven for Apache Kafka®
+=====================================================
+Learn how to enable tiered storage capability of Aiven for Apache Kafka®. This topic provides step-by-step instructions for maximizing storage efficiency using either the `Aiven console <https://console.aiven.io/>`_ or the :doc:`Aiven CLI </docs/tools/cli>`.
+
+.. important:: 
+    
+   Aiven for Apache Kafka® tiered storage is an early availability feature, which means it has some restrictions on the functionality and service level agreement. It is intended for non-production environments, but you can test it with production-like workloads to assess the performance. To enable this feature, navigate to the :doc:`Feature preview </docs/platform/howto/feature-preview>` page within your user profile.
+
+Prerequisites
+--------------
+* Aiven account and a project set up in the Aiven Console
+* Aiven for Apache Kafka® service with Apache Kafka version 3.6
+* Aiven CLI
+
+
+Enable tiered storage via Aiven Console
```

Comment:
Should we mention the prerequisites? Kafka 3.6 and must be supported in a region. I wonder if we could pull a list of the supported regions so user does not have to guess 🤔

Reply:
I have the prerequisites at the begining of the topic. As for a list of plans, it is hard to mention all the details in the docs. However, I can cross link the [plans and pricing page ](https://aiven.io/pricing?product=kafka) from the website, not sure how useful this will be. We don't mention all the versions and plans/regions in the docs.

Reply:
I have the prerequisites at the begining of the topic. As for a list of plans, it is hard to mention all the details in the docs. However, I can cross link the [plans and pricing page ](https://aiven.io/pricing?product=kafka) from the website, not sure how useful this will be. We don't mention all the versions and plans/regions in the docs.

Reply:
I have the prerequisites at the beginning of the topic. As for a list of plans and supported regions, mentioning all the details in the docs seems complicated. However, I can cross-link the [plans and pricing page ](https://aiven.io/pricing?product=kafka) from the website; I wonder how useful this will be. We don't mention all the versions and plans/regions in the docs.